wiki/osu!supporter/id.md

```diff
@@ -1,55 +1,156 @@
+- Klik ikon pensil yang terletak pada pojok kanan bawah sampul profil.
+- Pilih gambar sampul yang dikehendaki dari pilihan yang tersedia, atau unggah gambar sampulmu sendiri (ukuran yang disarankan adalah 2400x640).
 
-![Kartu pengguna pemilik osu!supporter](img/usercard.png?2 "Kartu pengguna pemilik osu!supporter")
+Setelah masa berlaku osu!supporter berakhir, gambar sampul ini akan tetap tertera pada laman profil yang bersangkutan meskipun tidak akan lagi dapat diubah.
 
-Di web osu!, lencana osu!supporter dapat ditemukan di sebelah avatar pengguna dan di kartu pengguna di samping bendera negara Anda.
+### Hak untuk mengganti nama pengguna secara cuma-cuma
 
-### Memeriksa status osu!supporter Anda
+*Catatan: Penggantian nama pengguna yang bersifat minor/sepele dapat diajukan kepada account support team kami secara cuma-cuma terlepas dari status supporter yang kamu miliki — untuk keterangan lebih lanjut, lihat [Pusat Bantuan/Akun § Apakah saya dapat mengganti nama pengguna yang saya miliki?](/wiki/Help_centre/Account#name-changes).*
 
-![osu!supporter tag yang berdurasi 3 bulan tersisa](img/status.jpg?1 "Tiga bulan tersisa dari osu!supporter tag")
+Para supporter yang belum pernah mengganti nama penggunanya akan dapat mengganti nama pengguna mereka untuk pertama kalinya secara cuma-cuma sesuai dengan [ketentuan yang berlaku](/wiki/Help_centre/Account#name-changes).
 
-*Catatan: Bagian ini hanya ditampilkan setelah Anda memiliki osu!supporter.*
+### osu!direct
 
-Untuk memeriksa status osu!supporter tag Anda yang tersisa, total kontribusi, dan pembelian tag, buka halaman [dukung permainan ini](https://osu.ppy.sh/home/support) dan scroll ke bawah untuk melihat bagian yang serupa dengan gambar di atas.
+osu!direct merupakan fitur pencarian dan penguduhan beatmap kilat yang khusus tersedia bagi para supporter. Fitur ini dapat diakses melalui tombol `osu!direct` yang tertera pada sisi kanan layar menu utama. Di samping akses langsung menuju daftar beatmap, osu!direct juga menyediakan berbagai cara lainnya untuk dapat memperoleh beatmap tanpa harus meninggalkan permainan sebagai berikut:
+
+- Tautan beatmap pada *chat*, yang akan menampilkan dialog unduhan dalam permainan
+- Pengunduhan beatmap otomatis dalam lobi [multiplayer](/wiki/Gameplay/Multiplayer) atau mode [spectate](/wiki/Gameplay/Spectating), yang dapat [dinonaktifkan kapanpun](/wiki/Client/Options#integration)
+- Penautan beatmap yang sedang dimainkan secara otomatis pada kanal `#spectator`, yang dapat [dinonaktifkan kapanpun](/wiki/Client/Options#integration)
+
+### Papan peringkat tambahan
+
+Para supporter akan dapat mengakses berbagai [papan peringkat](/wiki/Beatmap#papan-peringkat) khusus pada masing-masing beatmap, baik di dalam permainan maupun melalui situs web osu!, sebagai berikut:
+
+- Papan peringkat global untuk kombinasi [mod permainan](/wiki/Game_modifier) tertentu yang dikehendaki
+- Papan peringkat negara yang mencakup negara pemilik supporter
```

Comment:
```suggestion
- Papan peringkat negara dengan bendera yang sama
```

Reply:
`pemilik supporter` sound like the owner of it. since this one user experience, leaving without it is fine.